File: src/nomanssave/r.java
package nomanssave;

import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;
import java.util.ArrayList;

class r implements ActionListener {
   // $FF: synthetic field
   final p I;

   r(p var1) {
      this.I = var1;
   }

   public void actionPerformed(ActionEvent var1) {
      int[] var2 = p.b(this.I).getSelectedRows();
      p.a(this.I, new ArrayList());

      for(int var4 = 0; var4 < var2.length; ++var4) {
         int var3 = p.b(this.I).convertRowIndexToModel(var2[var4]);
         p.c(this.I).add((String)p.b(this.I).getModel().getValueAt(var3, 3));
      }

      this.I.setVisible(false);
   }
}

File: NMSSaveEditor/nomanssave/lower/r.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace NMSSaveEditor
{

class r : EventHandler {
   p I;

   r(p var1) {
      this.I = var1;
   }

   public void actionPerformed(EventArgs var1) {
      int[] var2 = p.b(this.I).getSelectedRows();
      p.a(this.I, new List<object>());

      for(int var4 = 0; var4 < var2.Length; ++var4) {
         int var3 = p.b(this.I).convertRowIndexToModel(var2[var4]);
         p.c(this.I).Add((string)p.b(this.I).getModel().getValueAt(var3, 3));
      }

      this.I.setVisible(false);
   }
}

}
